lib/Controller/SettingsController.php

```diff
@@ -204,6 +204,24 @@ public function setSettings($wopi_url,
 
 		return new JSONResponse($response);
 	}
+	/**
+	 * @NoAdminRequired
+	 * @throws \Exception
+	 * @return JSONResponse
+	 */
+	public function checkWebServer() {
+		try {
```

Comment:
Maybe you could move the SERVER_SOFTWARE handover to https://github.com/nextcloud/richdocuments/blob/0bd69be42b242b651c25410d0cb6646042b7ff81/lib/Settings/Admin.php#L92 You can directly access it then in the AdminSettings.vue though this.settings without the need for an additional request.